prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/MN/MN-receiver-transform.yml

```diff
@@ -0,0 +1,74 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+constants:
+  PID: '/PATIENT_RESULT/PATIENT/PID'
+  SPM: '/PATIENT_RESULT/ORDER_OBSERVATION/SPECIMEN/SPM'
+
+elements:
+
+  - name: xtn-use-code-from-system
+    condition: '%resource.system = "email"'
+    value: [ '"NET"' ]
+    hl7Spec: [ '%{hl7TelecomPath}-2' ]
+
```

Comment:
@oslynn Why are you include this for every STLT why is this needed?

Reply:
Ok, if we don't need it, I will delete it.